### DISCUSSION TITLE
Linux基金会企业开源指南系列之八——公司如何正确的使用开源代码 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#89] [Announcements]
# Linux基金会企业开源指南系列之八——公司如何正确的使用开源代码 - 「开源之道」

https://opensourceway.community/posts/opensource_enterprise_guide/using-open-source/

**Comment by whybeyoung** [comment]
翻译者头部这句话 ,略显极端

**Reply by lijiangsheng1** [reply]
是的，2018年时的我，对于中国的法律情况还理解颇浅，又经过了5年，其实，我还是保守了。现实比这个更严重。
